projects/batfish/src/main/antlr4/org/batfish/grammar/flatjuniper/FlatJuniper_bgp.g4

```diff
@@ -421,11 +482,39 @@ bps_external_router_id
    EXTERNAL_ROUTER_ID
 ;
 
+b_secret_key
+:
+   SECRET key = string_value
+;
+
+b_start_time
+:
+   START_TIME time = variable
+;
+
+b_tolerance
+:
+   TOLERANCE value = DEC
+;
+
+string_value
+:
+   (
+      VARIABLE
+      | NON_WHITE_SPACE_STRING
+   )
+   |
+   (
+      DOUBLE_QUOTED_STRING
+   )
+;
```

Comment:
why is the rule for string_value not simply: 
( 
    DOUBLE_QUOTED_STRING 
    | NON_WHITE_SPACE_STRING 
    | VARIABLE
)

One group and alphabetical.